docs/webpack.config.js

```diff
@@ -26,14 +26,19 @@ module.exports = {
           {
             loader: 'stylus-loader',
             options: {
-              use: [ require('../stylus')() ]
+              use: [require('../stylus')()]
             }
           }
         ]
       }
     ]
   },
   plugins: [
-    new MiniCssExtractPlugin('[name].css')
+    new MiniCssExtractPlugin('[name].css'),
+    new webpack.DefinePlugin({
+      'process.env': {
+        USE_REACT: 'true'
```

Comment:
Is it intended ? It means that we use react now in the styleguidist right ?

Reply:
The doc was not working correctly with portals since styleguidist uses React but without this variable it was using `preact-portal` for portals :/